team-manager/src/test/unit/java/ua/knu/knudev/teammanager/service/DepartmentServiceTest.java

```diff
@@ -151,4 +157,163 @@ void should_throwDepartmentException_When_DepartmentDoesNotExistOnGet() {
 
         verify(departmentRepository, times(1)).findById(TEST_DEPARTMENT_ID);
     }
+
+    @Test
+    @DisplayName("Should create department when given valid data")
+    public void should_createDepartment_When_givenValidData() {
+        SpecialtyCreationDto specialtyCreationDto1 = new SpecialtyCreationDto(14.378, "Computer Engineering");
```

Comment:
Put here Arrange, Act, Assert comments for better readability, because test is kind of large

Reply:
done